README.md

```diff
@@ -40,6 +40,32 @@ When node starts, the ids of node, store and region must be registered into pd a
 
 A store starts up a rocksdb. Store also record its meta and raft group information in local as well as pd has managed them all via etcd. 
 
+### Install Tips
+
+```
+# 1. Install rust nightly.
+curl -s https://static.rust-lang.org/rustup.sh | sh -s -- --channel=nightly
+
+# 2. Install RocksDB
+# Please view RocksDB's offical install document at https://github.com/facebook/rocksdb/blob/master/INSTALL.md . It's important to note, upgrade your GCC to version at least 4.7 to get C++11 support.
+wget https://github.com/facebook/rocksdb/archive/rocksdb-4.3.1.tar.gz
+tar -xzvf rocksdb-4.3.1.tar.gz
+
+# [optional step] clone the newest version of RocksDB
+git clone https://github.com/facebook/rocksdb.git
+
+cd rocksdb
+make shared_lib
+cp librocksdb.so* /usr/lib64/
```

Comment:
no need to copy rocksdb header file to /user/include ?

btw, I prefer /usr/local/lib, Mac has no `/usr/lib64` path. :-)